src/cpp/src/image_generation/stable_diffusion_3_pipeline.hpp

```diff
@@ -397,17 +416,42 @@ class StableDiffusion3Pipeline : public DiffusionPipeline {
         m_transformer->set_hidden_states("pooled_projections", pooled_prompt_embeds_inp);
     }
 
+    std::vector<float> get_timesteps(size_t num_inference_steps, float strength) {
```

Comment:
Why is it not part of `Scheduler`? 
`set_timesteps` accepts `strength` parameter and can patch returned timesteps

The same for FLUX

Reply:
Was done similar to https://github.com/huggingface/diffusers/blob/9a147b82f72e5df4553cb0f845bb957be3aa6028/src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3_inpaint.py#L671

This method doesn't return the Scheduler timesteps, but rather selects the required set of timesteps for img2img generation, I don’t think this should be part of the Scheduler